### PR TITLE
Update for docker client in mapapp

### PR DESCRIPTION
**Pull request by cbautista1002**
This PR adds updates that are needed to allows MAP to manage docker build-test-publish cycles:
- Install docker in tacstack mapapp image
- Clone docker-t7 github repo in `bdt_map_setup.sh` and `bdt_map_testenv_setup.py`
- Add `DOCKER_HOST` envvar to `.bashrc`
- Set 755 permissions for 3 setup scripts in `app/src/home`

### CODE REVIEW

**Comment by cbautista1002** [comment]
@isehgu @jumanjiman @gganeshan Please review and/or merge

**Comment by cbautista1002** [comment]
We're still in POC mode for this so hardcoding `DOCKER_HOST` in `.bashrc` is ok for now. Once this process matures we'll update this. You guys ok with that?

File: app/src/root/.bashrc [hunk 1]
```diff
@@ -27,3 +27,4 @@ alias launchmap='startmap; tml'
 export TERM=xterm
 cd /home
 
+export DOCKER_HOST=tcp://td-lss01.test.ise.com:2375
```
Comment (jumanjiman) [inline app/src/root/.bashrc]: it'd be much better to do:

```
# use DOCKER_HOST env var if it exists; fall through to td-lss01 by default
docker_host=${DOCKER_HOST:-"tcp://td-lss01.test.ise.com:2375"}
export DOCKER_HOST=$docker_host
```

Comment (cbautista1002) [inline app/src/root/.bashrc]: Very nice :+1:

Comment (jumanjiman) [inline app/src/root/.bashrc]: bash expansion ftw!

**Comment by cbautista1002** [comment]
Updated with check for `DOCKER_HOST` value before falling back to default.

**Comment by jumanjiman** [comment]
https://circleci.com/gh/ISEexchange/tacstack/101 is clean

will :ship:

**Comment by cbautista1002** [comment]
Thanks!

**Comment by jumanjiman** [comment]
merged!

**Pull request merged**